src/main/java/uk/ac/ebi/eva/contigalias/entities/SequenceEntity.java

```diff
@@ -16,42 +16,38 @@
 
 package uk.ac.ebi.eva.contigalias.entities;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import io.swagger.annotations.ApiModelProperty;
 
 import javax.persistence.CascadeType;
 import javax.persistence.Column;
 import javax.persistence.EnumType;
 import javax.persistence.Enumerated;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.ManyToOne;
 import javax.persistence.MappedSuperclass;
 
 @MappedSuperclass
 public class SequenceEntity {
 
-    @JsonIgnore
-    @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    @Column(nullable = false)
-    @ApiModelProperty(value = "Autogenerated Primary Key.")
-    private long id;
-
     @ApiModelProperty(value = "GenBank's name of the sequence.")
     private String genbankSequenceName;
 
     @ApiModelProperty(value = "ENA's name of the sequence")
     private String enaSequenceName;
 
-    @ApiModelProperty(value = "Sequence's GenBank accession.")
-    private String genbank;
+    @Id
+    @Column(nullable = false)
+    @ApiModelProperty(value = "Sequence's INSDC accession.")
+    private String insdcAccession;
 
+    @Column(nullable = false)
     @ApiModelProperty(value = "Sequence's RefSeq accession.")
     private String refseq;
 
+    @ApiModelProperty(value = "Sequence's length")
+    private Long seqLength;
```

Comment:
Probably just some context I missed, but why are we adding sequence length?  Is it just another useful property to have or is there another reason?

Reply:
Suggested by Tim to include as he generally uses it for comparing